src/buildercore/cfngen.py

```diff
@@ -400,6 +400,9 @@ def _parameterize_gcslogging(gcslogging):
             'dns': pdata['aws']['fastly']['dns'],
             'default-ttl': pdata['aws']['fastly']['default-ttl'],
             'healthcheck': pdata['aws']['fastly']['healthcheck'],
+            'between-bytes-timeout': pdata['aws']['fastly'].get('between-bytes-timeout'),
```

Comment:
these are all options to be applied to all backends (if multiple ones are defined); so the level of nesting seems ok. The naming should be kept as Fastly defines these, with no translation